src/components/Modals/Modal.vue

```diff
@@ -131,12 +132,16 @@ export default {
       fetch(`https://classes.cornell.edu/api/2.0/search/classes.json?roster=${roster}&subject=${subject}&q=${courseCode}`)
         .then(res => res.json())
         .then(resultJSON => {
-          const course = resultJSON.data.classes[0];
-          course.roster = roster;
-
-          if (this.courseIsAddable) {
-            parent.addCourse(course);
-          }
+          // check catalogNbr of resultJSON class matches number of course to add
+          resultJSON.data.classes.forEach(resultJSONclass => {
```

Comment:
A lot of this code is on Theresa's branch (which has not yet been approved). This should not be merged in until then.

Reply:
I git merge from Theresa's branch (I didn't rewrite everything) because there were several big bugs that I wanted to be addressed so the code should be the same. If we merge in everything atm (granted that Theresa hasn't made big changes in her branch since), there shouldn't be a problem if we merge it in at the right order

Reply:
In hindsight, merging with an incompleted branch is a bad idea. Sorry about the inconvenience that this might cause. Won't happen again